eng/common/templates/job/job.yml

```diff
@@ -132,7 +132,7 @@ jobs:
       displayName: RichCodeNav Upload
       inputs:
         languages: 'csharp'
-        environment: ${{ coalesce(parameters.richCodeNavigationEnvironment, 'prod') }}
+        environment: ${{ coalesce(parameters.richCodeNavigationEnvironment, 'production') }}
```

Comment:
All the changes under `eng/common/` need to be done in dotnet/arcade repository directly if not they will be overridden by our bot.

Reply:
Definitely! Sorry, I should have linked to [this PR](https://github.com/dotnet/arcade/pull/6345) - I only included them here as well to test.

Reply:
I went ahead and removed this - there's enough things to verify with your pipelines that I think it's worth just waiting for the official merge